src/components/ScheduleDeactivationForm/ScheduleDeactivationForm.tsx

```diff
@@ -252,7 +253,9 @@ export const ScheduleDeactivationForm = ({
                     ))}
 
                     <StyledFormInput
-                        defaultValue={scheduledDeactivation?.deactivateDate.toISOString().split('T')[0]}
+                        value={
+                            watch('deactivateDate') ? watch('deactivateDate').toISOString().split('T')[0] : undefined
```

Comment:
Every watch call creates subscription. You can call it once and save in a variable